packages/vat-data/src/index.js

```diff
@@ -46,3 +46,11 @@ export const {
   makeScalarBigSetStore,
   makeScalarBigWeakSetStore,
 } = VatDataGlobal;
+
+/**
+ * @type {import('./types').PickFacet}
+ */
+export const pickFacet =
```

Comment:
Please add a descriptive comment of what this is for.

Reply:
<img width="926" alt="Screen Shot 2022-04-12 at 2 08 38 PM" src="https://user-images.githubusercontent.com/21505/163055359-a22ec79f-8837-4382-aba4-bc4e0d4782ad.png">